mintpy/prep_aria.py

```diff
@@ -267,22 +280,26 @@ def write_ifgram_stack(h5File, unwStack, cohStack, connCompStack):
 
         h5["date"][ii,0] = d12.split("_")[0].encode("utf-8")
         h5["date"][ii,1] = d12.split("_")[1].encode("utf-8")
+        h5["dropIfgram"][ii] = True
 
         bnd = dsUnw.GetRasterBand(bndIdx)
-        h5["unwrapPhase"][ii,:,:] = -1.0*bnd.ReadAsArray()
+        data = bnd.ReadAsArray()
+        data[data == noDataValueUnw] = 0      #assign pixel with no-data to 0
+        h5["unwrapPhase"][ii,:,:] = -1.0*data #date2_date1 -> date1_date2
```

Comment:
@yunjunz just double checking if we are in sync.

For ARIA standard product:
Reference (master) is acquired later in time than secondary (slave). 
Product sign convention: land uplift corresponds to a range decrease (negative radians).

Reply:
@yunjunz can confirm better.
In Mintpy outputs, uplift is positive and subsidence is negative (consistent with ARIA). The -1 here is because of the date swap. Based on our offline discussions, ARIA products always have date2_date1 form while Mntpy expects the opposite. The datelist datset is sorted as MintPy expects so we put -1 here.

Reply:
MintPy convention: land uplift (moving towards satellite) corresponds to negative phase value in radian. It seems the same...

Could you check your aria+mintpy result on Mauna Loa (Sen desc track 87 IW2) with the one below? It should be inflating on the right side (black cross in the map).
https://insarmaps.miami.edu/start/19.7/-155.6/7.5?onlyShowDatasets=S1_IW2_087_0528_0532_20141116_XXXXXXXX,&zoomOut=false

<img width="1278" alt="Screen Shot 2019-07-26 at 10 07 47 PM" src="https://user-images.githubusercontent.com/13143710/61988629-d78f9080-aff1-11e9-87f7-26bf5e3ed1fe.png">

Reply:
@yunjunz from the figure you posted, uplift (right side red signal) is positive. Isn't it?

Reply:
@hfattahi yes, it is positive as phase (displacement) in meters; and mintpy expects an equivalent interferogram with negative unwrapped phase in radian. At the end of invert_network step, mintpy multiply all phase_in_radian by ( wavelength / (-4pi)).

Reply:
That's right and I was referring to MintPy outputs.

Reply:
The sign of the final products in the San Fran dataset makes sense with MintPy convention. If @dbekaert confirms the sign in the final MintPy product of Hawaii dataset, then we should be good to merge this PR.

Reply:
I concur same sign for Hawaii.  I made it a while ago after @hfattahi made a change on the sign per Brett's request to make it work  for Norfolk. I do not think it has been changed since then.

Reply:
I guess that’s enough for confirm the sign is right.